README.md

```diff
@@ -22,9 +22,17 @@
 We're working on updating our websites, domains, and npm package names so that it will be easier in future to find all things Primer. We welcome feedback and contributions at any time. Please check existing issues and pull requests before contributing, and please communicate via an issue first if you intend to make large-scale changes. See [contributing](.github/CONTRIBUTING.md) for more info.
 
 #### Team
+- [@pouretrebelle](https://github.com/pouretrebelle)
```

Comment:
thanks for updating this 🙏

Reply:
👍 Added it to onboarding project templates